isso/ext/notifications.py

```diff
@@ -175,12 +176,17 @@ def notify_users(self, thread, comment):
 
     def sendmail(self, subject, body, thread, comment, to=None):
         to = to or self.conf.get("to")
-        if uwsgi:
+        if self.isso.uwsgi and uwsgi:
```

Comment:
This should not be necessary; uwsgi.spool should work in all cases, but you need to have a spooler configured in your uwsgi.ini.